src/installer/corehost/cli/hostmisc/pal.unix.cpp

```diff
@@ -13,9 +13,12 @@
 #include <fcntl.h>
 #include <fnmatch.h>
 #include <ctime>
+#include <clocale>
```

Comment:
Can you please make sure that this change doesn't regress the size of apphosts? In theory it should not as the new functionality is not used by apphosts, but still.

Reply:
On Linux X64 Release build, apphost binary included the `unicode_palstring` function -- so it wasn't removed by the compiler. But the apphost size only increased by 16 bytes (the function's size is definitely bigger than 16 bytes).